pkg/collector/corechecks/system/filehandles/file_handles_windows.go

```diff
@@ -56,6 +58,16 @@ func (c *fhCheck) Configure(data integration.Data, initConfig integration.Data,
 		return err
 	}
 
+	// Create PDH query
+	c.pdhQuery, err = pdhutil.CreatePdhQuery()
+	if err != nil {
+		return err
+	}
+
+	c.counters = map[string]pdhutil.PdhSingleInstanceCounter{
```

Comment:
I am torn if we need map. Not now but the moment we need to add more counters it could be useful

Reply:
I feel the same. I figured I'd add it now so the next person could add onto it rather than adding another variable like all the other checks were before.